test/PensionFundReleaseTest.js

```diff
@@ -1,87 +1,178 @@
+      let validator = await fund.validators.call(0)
+      validator.should.equal(VALIDATORS[0])
+    })
+    
+    it("#3 should allow VALIDATORS to vote", async () => {
+      await fund.vote(true, "justification", {from: VALIDATORS[0]})
+      let index = await fund.voteIndex.call(VALIDATORS[0])
+      let vote = await fund.votes.call(index)
+      vote[2].should.be.equal("justification")
+    })
+
+    it("#4 should not allow non-VALIDATORS to vote", async () => {
+      fund.vote(true, "justification", {from: UNAUTHORIZED}).should.be.rejected
+    })
+
+    it("#5 should allow release after all VALIDATORS approval", async () => {
+      await fund.vote(true, "justification", {from: VALIDATORS[0]})
+      await fund.vote(true, "justification", {from: VALIDATORS[1]})
+      let approval =  await fund.isReleaseApproved.call()
+      approval.should.be.equal(true)
+    })
+
+    it("#6 should not allow release if not all VALIDATORS approved release", async () => {
+      await fund.vote(true, "justification", {from: VALIDATORS[0]})
+      let vote = await fund.isReleaseApproved.call()
+      vote.should.be.equal(false)
+    })
+
+    it("#7 should allow burn after all VALIDATORS rejection", async () => {
+      await fund.vote(false, "justification", {from: VALIDATORS[0]})
+      await fund.vote(false, "justification", {from: VALIDATORS[1]})
+      let approval = await fund.isBurnApproved.call()
+      approval.should.be.equal(true)
+    })
+
+    it("#8 should not allow burn if not all VALIDATORS rejected release", async () => {
+      await fund.vote(false, "justification", {from: VALIDATORS[0]})
+      let approval = await fund.isBurnApproved.call()
+      approval.should.be.equal(false)
+    })
+
+    it("#9 should release roots if all conditions met", async () => {
+      let workerBalance, balanceNumber
+      let balance = INITIAL_BALANCE      
+      await token.transfer(fund.address, balance)
+      await fund.vote(true, "justification", {from: VALIDATORS[0]})
+      await fund.vote(true, "justification", {from: VALIDATORS[1]})
+      let release = await fund.releaseRoots({from: WORKER})
+      release.logs[0].args.amount.toNumber().should.be.equal(balance* 20 / 100)
+      workerBalance = await token.balanceOf(WORKER)
+      workerBalance.toNumber().should.be.equal(balance * 20 / 100)
+    })
+
+    it("#10 should attempt to release roots and fail", async () => {
+      let balance = INITIAL_BALANCE
+      fund = await PensionFundRelease.new.apply(
+        this,
+        deployParams(
+          firstPaymentTime + TIME_INCREMENT,
+          FIRST_PAYMENT_PERCENT,
+          token.address,
+          PAYOUT_PERCENT
+        )
+      )         
+      await token.transfer(fund.address, balance)
+      await fund.vote(true, "justification", {from: VALIDATORS[0]})
+      await fund.vote(true, "justification", {from: VALIDATORS[1]})
+      try {
+        await fund.releaseRoots({from: WORKER})
```

Comment:
Alright yeah I just tested it and ``fund.releaseRoots({from: WORKER}).should.be.rejected()`` gives the error ``TypeError: fund.releaseRoots(...).should.be.rejected is not a function``. Similar things happen if I include await or try ``fund.releaseRoots({from: WORKER}).should.eventually.be.rejected()``

For now the solution I have is:
```      try {
        let release = await fund.releaseRoots({from: WORKER})
      } catch (err) {
        error = err
      } 
      error.should.be.a("Error")```
I think long term it would be good to consider either reworking that helper or using it. What do you think?